app/views/hyrax/base/_form_visibility_error.html.erb

```diff
@@ -1,4 +1,3 @@
 <%= f.full_error(:visibility) %>
 <%= f.full_error(:embargo_release_date) %>
 <%= f.full_error(:visibility_after_embargo) %>
-<%= f.full_error(:single_collection) %>
```

Comment:
This error should not be mixed in with visibility related errors.  Moved to _form_collections_error.html.erb which will hold all collection related errors.